GitUI/Theme/dark.thm

```diff
@@ -0,0 +1,41 @@
+ActiveBorder: 5e6060
```

Comment:
Can we have `.theme` extension?

Reply:
.theme is used by windows theme

Reply:
changed to .colors

Reply:
> .theme is used by windows theme

It doesn't stop us from using the same 😉 



> changed to .colors

Works for me